pkg/generator/generator.go

```diff
@@ -32,6 +32,7 @@ type Config struct {
 	ReleaseImageMirror string
 	DummyIgnition      bool   `envconfig:"DUMMY_IGNITION"`
 	InstallInvoker     string `envconfig:"INSTALL_INVOKER" default:"assisted-installer"`
+	IcspFile           string `envconfig:"ICSP_FILE" default:""`
```

Comment:
Who should set this env?
What should be the content of `IcspFile` is it the imageContentSourcePolicy content or file path?

Reply:
This will be set when running BILLI

Reply:
Yes this will be set when running BLLI, it will be the file path of a mounted ICSP API file.

Reply:
We are currently passing into to the assisted-service container the registries.conf file, mounted in /etc/containers, and this is used [here](https://github.com/openshift/assisted-service/blob/dd79548c59fb0756b5a6536a8868ab29123dfcd7/internal/installcfg/builder/builder.go#L137)
to set up the ImageContentSources in the install-config. We could also use that to build up an ImageContenSourcePolicy [API](https://docs.openshift.com/container-platform/4.10/rest_api/operator_apis/imagecontentsourcepolicy-operator-openshift-io-v1alpha1.html) to pass into the 'oc' command if that is the preferred way to go.

Reply:
This should also be set in MCE. Doing this is a pre-requisite to switching to the official `oc` command instead of using a fork.
BILLI is the first user only because it has to use official builds.